AKCOO/Scene/ExpenseScene/View/budgetCategorySettingView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  budgetCategorySettingView.swift
+//  AKCOO
+//
+//  Created by 이연정 on 11/9/24.
+//
+
+import UIKit
+
+class BudgetCategorySettingView: UIView {
+  
+  // MARK: - Views
+  private let categoryLabel = UILabel().set {
+    $0.text = "카테고리"
+    $0.font = UIFont.akFont(.gmarketMedium16)
+    $0.adjustsFontForContentSizeCategory = true
+    $0.textColor = UIColor.akColor(.gray3)
+  }
```

Comment:
`$0.adjustsFontForContentSizeCategory = true` 코드 위치를 최하단으로 두면 어떨까요? 통일성을 갖추기 위함입니다 ! 
```suggestion swift 
  private let categoryLabel = UILabel().set {
    $0.text = "카테고리"
    $0.font = UIFont.akFont(.gmarketMedium16)
    $0.textColor = UIColor.akColor(.gray3)
    $0.adjustsFontForContentSizeCategory = true
  }
```